src/distributions/adopt/installer.ts

```diff
@@ -70,6 +77,15 @@ export class AdoptDistribution extends JavaBase {
     return { version: javaRelease.version, path: javaPath };
   }
 
+  protected get toolcacheFolderName(): string {
+    if (this.jvmImpl === AdoptImplementation.Hotspot) {
```

Comment:
Should be `AdoptImplementation.Hotspot.toLowerCase()`
Sorry for providing invalid example

Reply:
right, no problem. I'll fix it

Reply:
Looks like PR checks hang for some reason 😄 
I will validate caching as soon as they wake up.